### PR TITLE
pyenv 1.2.14

### PR DESCRIPTION
**Pull request by joshfriend**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/pyenv.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Pyenv < Formula
   desc "Python version management"
   homepage "https://github.com/pyenv/pyenv"
-  url "https://github.com/pyenv/pyenv/archive/v1.2.13.tar.gz"
-  sha256 "ebf9899f70cb04a6a6bf9835c37d9d7e4ed7dadb22dd8123b19d6d790a13fffe"
+  url "https://github.com/pyenv/pyenv/archive/v1.2.14.tar.gz"
+  sha256 "3062c104b200d8c572d185b54e73a94bf66d5d46cc789717c372d2941c314a93"
   revision 1
```
Comment (chenrui333) [inline Formula/pyenv.rb]: Can you remove `revision` and `version_scheme` lines? Thanks!

Comment (joshfriend) [reply Formula/pyenv.rb]: Done!

**Comment by tedmiston** [comment]
Are any changes required to get this merged?  I'm really looking forward to trying Python 3.8 with pyenv.

**Review by chenrui333 — APPROVED** [review]
I will take care of it once it passed the CI. :)

**Comment by chenrui333** [comment]
@tedmiston, it will come shortly.

@joshfriend, thanks for taking care of the upgrade!

**Comment by chenrui333** [comment]
Enjoy the new pyenv release :)

**Pull request closed without merging**